pages/system/inputs.js

```diff
@@ -30,37 +29,46 @@ export default class SystemPageInputs extends React.Component {
         <System.P>The Input component is used to get a users input in a text field or a textbox.</System.P>
         <br />
         <br />
-        <System.H2>Usage</System.H2>
+        <br />
+        <System.H2>Imports</System.H2>
         <hr />
         <br />
-        <System.P>Define the Input value states and handle the state change the when a change is made.</System.P>
+        <System.P>
+          Import React and the Input and/or the Textarea Components.
+        </System.P>
+        <br />
         <br />
         <System.CodeBlock>
-          {`state = {
-  twelve: 'Replace me friend.',
-  thirteen: '',
-  fourteen: '',
-  fifteen: 'aaaaa-bbbbb-ccccc-ddddd-eeee',
-  sixteen: '',
-  seventeen: 'Example text',
-};
-
-_handleChange = (e) => {
-  this.setState({ [e.target.name]: e.target.value });
-};`}
+{`import * as React from 'react';
+import { Input, Textarea } from 'slate-react-system';`}
         </System.CodeBlock>
         <br />
         <br />
-        <br />
         <System.H2>Textarea</System.H2>
         <hr />
         <br />
         <System.P>Declare the Textarea component.</System.P>
         <br />
-        <System.Textarea name="seventeen" value={this.state.seventeen} onChange={this._handleChange} />
+        <System.Textarea name="exampleOne" value={this.state.exampleOne} onChange={this._handleChange} />
         <br />
         <System.CodeBlock>
-          {`<System.Textarea name="seventeen" value={this.state.seventeen} onChange={this._handleChange} />`}
+          {`class ExampleTextarea extends React.Component {
+   state = { exampleOne: 'Example text' }
+
+   _handleChange = (e) => this._handleChange(
```

Comment:
This is not correct, you need

`_handleChange = e => this.setState({ [e.target.name]: e.target.value });`